Bottom: 4e37d1a24ca55fc17f8a096b813467c85eec8c90
Top:    03f8c6d00f7e5fd7b5f325c2c570a95caf0fa49c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 18:55:00 +0000

autoroute.c: Remove unnecessary call to ResetConnections()

This is not required to call CollectSubnets, (as the rats.c internal
function GatherSubnets() cleans the flag from objects before it uses
it).

We also use DRCFLAG to mark nets in CreateRouteData(), however:

1. We should be able to assume the DRCFLAG is clear on all objects at the
   point we un-necessary to reset it.

2. We call GatherSubnets() after we reset the DRCFLAG contents, and
   that leaves the objects in a clean state without that flag set.


---

diff --git a/src/autoroute.c b/src/autoroute.c
index d7d6aa1..80aad25 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -1024,7 +1024,6 @@ CreateRouteData ()
    * this saves on searching the trees to find the nets
    */
   /* use the DRCFLAG to mark objects as they are entered */
-  ResetConnections (false);
   Nets = CollectSubnets (false);
   {
     routebox_t *last_net = NULL;
